src/podping_hivewriter/podping_hivewriter.py

```diff
@@ -161,32 +161,31 @@ async def test_hive_resources(self):
                 "uuid": str(uuid.uuid4()),
                 "hive": str(self.lighthive_client.current_node),
             }
-
-            await self.send_notification(custom_json, STARTUP_OPERATION_ID)
-            settings = await self.settings_manager.get_settings()
-            logging.info(
-                f"Testing Account Resource Credits {settings.hive_operation_period}s"
+            op, size_of_json = await self.construct_operation(
+                custom_json, STARTUP_OPERATION_ID
```

Comment:
Note sure why this is working but this is an invalid type... you have construct_operation() requiring "HiveOperationId" but giving it a string.

Also, STARTUP_OPERATION_ID is just set to "_startup".  Is this what you intended to set as the operation_id?

Reply:
What I'd like is for all pings to start with pp_ or pplt_ for podping and podping livetest when we're testing.

What I really don't understand is how the Operation ID works with the GPRC or whatever that google compiler is.

So whatever the correct way to add _startup for this (now single) custom_json... that's what we should do.